frontend/public/components/factory/modal.tsx

```diff
@@ -44,21 +48,27 @@ export const createModalLauncher: CreateModalLauncher = (Component) => (props) =
     return (
       <Provider store={store}>
         <Router {...{ history, basename: window.SERVER_FLAGS.basePath }}>
-          <Modal
-            isOpen={true}
-            contentLabel={i18next.t('public~Modal')}
-            onRequestClose={_handleClose}
-            className={classNames('modal-dialog', props.modalClassName)}
-            overlayClassName="co-overlay"
-            shouldCloseOnOverlayClick={!props.blocking}
-            parentSelector={() => document.getElementById('modal-container')}
-          >
-            <Component
-              {...(_.omit(props, 'blocking', 'modalClassName') as any)}
-              cancel={_handleCancel}
-              close={_handleClose}
-            />
-          </Modal>
+          <DetectPerspective>
```

Comment:
The addition of these components broke integration tests because of the hooks used in the components.